release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -166,6 +166,11 @@ The following metrics are sent through telemetry.
 
 |===
 
+[id="ocp-4-12-nw-ingress-rules"]
```

Comment:
This needs the under AWS Load Balancer Operator (new section)

Reply:
Similar to the `ExternalDNS` section

Reply:
Addressed